package.json

```diff
@@ -9,7 +9,7 @@
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "check:lint": "eslint --max-warnings=0 --ignore-path .eslintignore ./src",
-    "check:prettier": "prettier --ignore-path .prettierignore --plugin-search-dir=. --end-of-line=auto --check ./src ",
+    "check:prettier": "prettier --check ./src",
```

Comment:
remove the ./src and test that lint staged only use prettier on files where u did git add